aiocoap/transports/udp6.py

```diff
@@ -299,9 +308,15 @@ async def _create_transport_endpoint(cls, sock, ctx: interfaces.MessageManager,
         return protocol
 
     @classmethod
-    async def create_client_transport_endpoint(cls, ctx: interfaces.MessageManager, log, loop):
+    async def create_client_transport_endpoint(cls, ctx: interfaces.MessageManager, log, loop, broadcast: bool = False):
         sock = socket.socket(family=socket.AF_INET6, type=socket.SOCK_DGRAM)
         sock.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_V6ONLY, 0)
+        if broadcast is True:
```

Comment:
This is redundant with the code in _create_transport_endpoint, which is called right after this.